packages/back-end/src/services/apiKey.ts

```diff
@@ -1,17 +1,19 @@
-import uniqid from "uniqid";
 import { ApiKeyModel } from "../models/ApiKeyModel";
-import md5 from "md5";
+import crypto from "crypto";
 
 export async function createApiKey(
-  organization: string,
+  orgId: string,
   environment: string,
   description?: string
 ): Promise<string> {
   const key =
-    "key_" + environment.substr(0, 4) + "_" + md5(uniqid()).substr(0, 16);
+    "key_" +
+    environment.substr(0, 4) +
+    "_" +
+    crypto.randomBytes(24).toString("hex").substr(0, 24);
```

Comment:
This is not very secure if we're going to be using this as a private key.  Hex needs twice as many characters so in reality, this is only using 12 random bytes.  For comparison, Stripe uses 64+ random bytes for their keys.  Also, we might want to use base64 encoding instead of hex which uses 33% less space (although we should remove `+` and `/` to make it URL safe).